grid/app/__init__.py

```diff
@@ -53,7 +53,7 @@ def set_database_config(app, test_config=None, verbose=False):
     db.init_app(app)
 
 
-def create_app(debug=False, n_replica=None, test_config=None):
+def create_app(node_id: str, debug=False, n_replica=None, test_config=None):
     """Create flask application."""
```

Comment:
Docstring need to be updated to add info about arguments 
```suggestion
    """Create flask application.

    Args:
        node_id (str): [description]
        debug (bool, optional): [description]. Defaults to False.
        n_replica ([type], optional): [description]. Defaults to None.
        test_config ([type], optional): [description]. Defaults to None.

    Returns:
        app (Flask): Flask application.
    """
```